nifi-commons/nifi-flow-encryptor/src/test/java/org/apache/nifi/flow/encryptor/StandardFlowEncryptorTest.java

```diff
@@ -89,7 +100,145 @@ public void testProcessNoEncrypted() {
+                "          \"sample_false\": false,\n" +
+                "          \"sample_empty_array\": [],\n" +
+                "          \"key-attribute-encoding\": \"utf-8\",\n" +
+                "          \"auto.offset.reset\": \"latest\"\n" +
+                "        }\n" +
+                "      }\n" +
+                "    ]\n" +
+                "  }", password);
+    }
+
+    private String getSampleFlowXml(final String password) {
+        Objects.requireNonNull(password);
+        return String.format("<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\"?>\n" +
+                "<!--SAMPLE_COMMENT-->\n" +
+                "<flowController encoding-version=\"1.0\">\n" +
+                "    <rootGroup>\n" +
+                "        <id>fcf146b2-0157-1000-7850-7adf1d31e3fa</id>\n" +
+                "        <name>NiFi Flow</name>\n" +
+                "        <position x=\"0.0\" y=\"0.0\"/>\n" +
+                "        <comment/>\n" +
+                "        <processGroup>\n" +
+                "            <id>8a61ec1d-0158-1000-3a1a-12c54fe77838</id>\n" +
+                "            <name>org.apache.nifi.processors.SAMPLE_PROCESSORS</name>\n" +
+                "            <position x=\"1119.0\" y=\"295.0\"/>\n" +
+                "            <comment/>\n" +
+                "            <processor>\n" +
+                "                <id>8a621f0b-0158-1000-b5c2-92a09a124501</id>\n" +
+                "                <name>Encrypt</name>\n" +
+                "                <position x=\"626.0\" y=\"237.0\"/>\n" +
+                "                <styles/>\n" +
+                "                <comment/>\n" +
+                "                <class>org.apache.nifi.processors.standard.EncryptContent</class>\n" +
+                "                <maxConcurrentTasks>1</maxConcurrentTasks>\n" +
+                "                <schedulingPeriod>0 sec</schedulingPeriod>\n" +
+                "                <yieldPeriod>1 sec</yieldPeriod>\n" +
+                "                <bulletinLevel>WARN</bulletinLevel>\n" +
+                "                <lossTolerant>false</lossTolerant>\n" +
+                "                <runDurationNanos>0</runDurationNanos>\n" +
+                "                <property>\n" +
+                "                    <name>Username</name>\n" +
+                "                    <value>SAMPLE_USERNAME</value>\n" +
+                "                </property>\n" +
+                "                <property>\n" +
+                "                    <name>Password</name>\n" +
+                "                    <value>%s</value>\n" +
+                "                </property>\n" +
+                "            </processor>\n" +
+                "        </processGroup>\n" +
+                "    </rootGroup>\n" +
+                "    <controllerServices/>\n" +
+                "    <reportingTasks/>\n" +
+                "</flowController>", password);
```

Comment:
Maintaining this much inline XML can be cumbersome, it is probably better to externalize this example, or use code to generate a simple XML string.